app/serializers/api/v1/order_serializer.rb

```diff
@@ -9,7 +9,7 @@ class OrderSerializer < ApplicationSerializer
       :dispatch_started_by_id, :submitted_at, :submitted_by_id
 
     has_one :created_by, serializer: UserProfileSerializer, root: :user
-    has_one :stockit_contact, serializer: StockitContactSerializer, root: :contact
+    has_one :stockit_contact, serializer: StockitContactSerializer
```

Comment:
@abulsayyad123  What was the reason for removing **root: :contact?**

Reply:
because the main contact serializer was getting blocked by stockit_contact root:contact.

Reply:
removing that is useful for GC orders but I think **StockItLocalOrders** might be using ** stockit_contact** . What will happen to LocalOrders on front-end because of this change?